Feature/ch3 alt recipes

ethanicusss opened the pull request:
Oh yeah baby.

There's a big wall of sequenced assembly at the start of Ch3, which I think may contribute to the burnout felt around here. This PR adds an alternate recipe for each of these. The original recipes have not changed.

Some are intended to be alternates for before automation is set up, and some are intended to be alternate options once there is solid steel production. Steel gets cheaper further into progression.

The original prec mech recipe is awesome, as it forces the player to look at better production of cogs, which does exist. The only annoying thing is the 80% chance that can't be countered. Now it can with zinc and steel.

May look at basic machine frames again, but probably in 2.2. It may also be nice to have steel variants of every casing.

**Basic Machine Frame** - the copper casing input can be exchanged for a steel frame. intended if the player doesn't have a good stock of copper casings
**Integrated Circuit** - manual craft. fairly long and diverse process, but not too expensive if you just want a few
**Precision Mechanism** - the cogs can be exchanged for their copycat counterparts (3 zinc), and the desh nugget is replaced with a steel ring (2 steel) for a 100% chance of a precision mechanism instead of the original 80%
**Brass Casing** - over double the cost but super convenient. apply a brass sheet to a copper casing in a smithing table, and then dump a bucket of brass over it in a casing basin

ethanicusss committed:
inefficient brass casing recipe

ethanicusss committed:
TEXTURES!!!!!

some temp

ethanicusss committed:
alternate integrated circuit recipe (manual)

manual craft for integrated circuit, for those who cba with looping assembly in ch3
requires ink sac, paper, copper sheet, lapis sheet, sulfur (3 glowstone, sky stone dust), water, oxygen
new recipe for sulfuric acid - sulfur, water and oxygen in a mixer. recipe less efficient than chemical reactor one

ethanicusss committed:
brass casing alt recipe change

swapped to remove a cheese

ethanicusss committed:
Merge branch 'Astral-Experimental' into feature/ch3-alt-recipes

ethanicusss committed:
alt precision mechanism recipe

1 brass, 3 zinc, 2 steel for a 100% chance of precision mechanism
steel rings are now 3 nuggets

ethanicusss committed:
Merge branch 'Astral-Experimental' into feature/ch3-alt-recipes

ethanicusss committed:
alternate basic machine frame

swap out the copper casing for a steel frame, costing 3 steel ingots. May be preferable if the player doesn't have a good backup of copper casing or plans to use a lot of them.

ethanicusss committed:
H2SO4 mixing fix

The-Shortman commented on kubejs/server_scripts/recipes/tconstruct/smeltery/casting_basin.js:
I'm a bit skeptical about 10 ingots per casing while the other recipe uses 4.8333... ingots per casing - this amounts to a recipe that's more than twice as expensive.

On that note looking into that horrible ratio for the existing brass casing recipe may be warranted at some point.

ethanicusss replied:
That's the idea. You use double the brass for a recipe that requires no prior setup, compared to the original recipe which requires a belt loop and brass in three different forms, but is cheaper.

Previously there's a "you must make belt loop" gate. With this PR, there's an alternative path, allowing the player to use brass casings without semi automation. Of course the player would still want the cheaper semi automation eventually, due to the many brass casings required afterwards for mechanical crafters.

KonSola5 commented:
> Brass Casing - over double the cost but super convenient. **apply a brass sheet** to a copper casing **in a smithing table**, and then dump a bucket of brass over it in a casing basin

I think this is a bit anti-Create here, as Create likes to use an in-world conversion or Deployer here.
Can this be changed to use the Manual Item Application, just like classic Create casings?

ethanicusss commented:
> > Brass Casing - over double the cost but super convenient. **apply a brass sheet** to a copper casing **in a smithing table**, and then dump a bucket of brass over it in a casing basin
> 
> I think this is a bit anti-Create here, as Create likes to use an in-world conversion or Deployer here. Can this be changed to use the Manual Item Application, just like classic Create casings?

Yeah good idea! I'll try something, though it might be hard to get the same brass amount

ethanicusss committed:
doncrete

make a new desh-based concrete block, which can be used to make a launchpad
probably needs balance
also desh dust and small desh dust
also reduced steel dust in concrete 3 -> 2

ethanicusss committed:
launchpad x2, minor recipe changes

ethanicusss committed:
alternate biofuel and slight buff to OG biofuel

compress 2 plantballs for 100% chance of 1 compressed plantball (OG is pressing for 80% chance)
wash compressed plantball for 80% chance of pure biomatter
extractor to get biomass from compressed plantball, for 100% chance washing or for bio pellets
nerfed energy usage of compressor compressed plantball -> biomass recipe massively

ethanicusss committed:
buff leather chemical reactor recipe

ethanicusss commented:
- blast-res cement now requires 2 steel instead of 3
 - alternate recipe for launchpad that uses Doncrete instead of Concrete. **Uses a stack of desh total and 8 buckets of lava**
 - both launchpad recipes now give 2 launchpads instead of 1
 - added Doncrete with an identical recipe as Concrete but 4 desh dust instead of 2 steel and 1/2 lava instead of water
 - alternate biofuel route:
 <img width="1427" height="743" alt="image" src="https://github.com/user-attachments/assets/b7b5b277-5df0-46f3-ac18-2d67c6e32548" />
**IMAGE NOT 100% ACCURATE**. The washing step gives a _chance_ of pure biomatter (80%), with a chance at getting a plantball (25%).

 - hugely reduced power consumption of the existing 4 compressed plantball -> biomass (compressing). 200E

 - introduced non-rng recipe for compressed plantball that uses compressor instead of mechanical press. 500E

 - also through the chemical reactor leather recipe buff in. 3 rotten flesh and 2 ashes for 5 leather. 900E

MikoTheBoi commented:
I feel like doncrete launchpad is way easier to make than blast-resistant (even if it gave only 1 launchpad). I came up with some ideas for the doncrete launchpad recipe: 

1. queen slime (2 launchpads) or slime blocks (because they're bouncy or smth) instead of sturdy sheets,
2. andesite iron billet instead of sturdy sheets, 
3. steel (this and billets less so since they still use iron), brass or electrum sheets instead of sturdy sheets (would only grant 1 pad this way),
4. Or if doncrete was made out of a desh+brass alloy?

Except for that I don't have any other comments!

ethanicusss committed:
alternate brass casing

ethanicusss committed:
extractor+grinder recipe buffs

couldn't remove
sturdy sheet -> gravel
or
lead sheet -> blue dye
dunno why

ethanicusss committed:
doncrete nerf

now takes 16 lava and 1.5 stacks of desh for the craft

ethanicusss committed:
and again

3 stacks of desh

The-Shortman commented:
Converting this to draft as it seems to still be in progress, please mark as ready for review once it's done

ethanicusss commented:
The changes I made were in response to the review from miko. I'm done now
